### ISSUE TITLE
SELECT * returns no rows, but SELECT tag, field returns something

### ISSUE THREAD

**Issue by johnp789** [influxdata/influxdb#7133]
### Bug report

__System info:__ 

Raspberry Pi 2, running Arch Linux ARM
InfluxDB 0.13.0, installed from AUR

__Steps to reproduce:__

1. Insert one record into a new table using CLI
2. Run SELECT * query
3. Run SELECT tag, field query

__Expected behavior:__ 

The two queries should return the same result, if all the tag and field keys are listed in the select clause of the second query.  

__Actual behavior:__ 

The SELECT * query does not return the row.  The SELECT tag, field query returns the one row that was inserted.  

```
$ cat input.txt 
use mon
INSERT toner_percent,printer=hpm402dn value=96i 1470792102000000000
INSERT page_count,printer=hpm402dn value=103i 1470792102000000000
$ cat input.txt | influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 0.13.0
InfluxDB shell version: 0.13.0
Using database mon
```

Then, in the influx CLI: 
```
$ influx
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 0.13.0
InfluxDB shell version: 0.13.0
> use mon
Using database mon
> show series
key
page_count,printer=hpm402dn
toner_percent,printer=hpm402dn

> SELECT * from page_count
> SELECT printer, value from page_count
name: page_count
----------------
time                    printer         value
1470792102000000000     hpm402dn        103
```

**Comment by johnp789** [comment]
Edit: 

I can only reproduce this if I insert a row into a new series, delete from series, then proceed as above, inserting the new row into the same series.

**Comment by MikeSchroll** [comment]
Feels like this may have been resolved in master. Ex: may be related to this fix: https://github.com/influxdata/influxdb/pull/7027

Should probably re-test on master or next release.

**Comment by jsternberg** [comment]
This sounds like what @MikeSchroll said and should be fixed in master or in 1.0.0-beta3.

**Comment by johnp789** [comment]
I can confirm that 1.0.0-beta3 fixes this issue on my system.  Thanks!
